src/pages/documentation/edge-network/release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## Feburary 13, 2023
+
+### Android Edge 1.4.1
+
+* Improves error handling when processing `Edge.sendEvent` requests by catching and logging `RuntimeException` instances thrown from the application provided `EdgeCallback` handler.
```

Comment:
Rewording suggestion: Improves error handling when processing `Edge.sendEvent` requests by catching and logging unexpected `RuntimeException` instances thrown from within caller's `EdgeCallback` handler implementation.